src/main/java/org/fogbowcloud/arrebol/execution/dockerworker/ContainerRequestHelper.java

```diff
@@ -51,11 +56,38 @@ private void post(String endpoint) throws Exception {
         this.httpWrapper.doRequest(HttpPost.METHOD_NAME, endpoint);
     }
 
-    private StringEntity jsonCreateContainer(String image) throws UnsupportedEncodingException {
+    public void setRequirements(Map<String, String> requirements) {
+        this.requirements = requirements;
+    }
+
+    protected void setImage(String image){
+        this.image = image;
+    }
+
+    private StringEntity jsonCreateContainer() throws UnsupportedEncodingException {
         JSONObject jsonObject = new JSONObject();
         AppUtil.makeBodyField(jsonObject, "Image", image);
         AppUtil.makeBodyField(jsonObject, "Tty", true);
+        //AppUtil.makeBodyField(jsonObject, "HostConfig", requirements);
```

Comment:
This field is really not required?

Reply:
Yes. When required the method "jsonAddRequirements" add the field.

Reply:
So, delete this line.